differentiable_blockchains.tex

```diff
@@ -93,7 +93,7 @@ \section{Consensus Protocols}
 \begin{equation} 
 \partial(\sigma) = \sum^{q+1}_{i=1} (-1)^{i-1}(\sigma \circ \delta_q^{i} )
 \end{equation}
-for continuous functions $\delta^{i}_q: \Delta^{q-1} \rightarrow \Delta^q | 1 \leq i \leq q+1$ where 
+for continuous functions $\delta^{i}_q: \Delta^{q-1} \rightarrow \Delta^q | 1 \leq i \leq q+1$ where % what is the "|" here?
```

Comment:
Is there a set being comprehended, or does 1 \leq i \leq q+1 go into a comment (e.g. info for a bracket), I wasn't sure but this looks unfamiliar.